lua/aerial/backends/lsp/callbacks.lua

```diff
@@ -88,7 +89,18 @@ local function process_symbols(symbols, bufnr, fix_start_col)
           if symbol.children then
             item.children = _process_symbols(symbol.children, item, {}, level + 1)
           end
-          table.insert(list, item)
+          if
+            config.post_parse_symbol == nil
```

Comment:
Because you're going to have to add the example to the `config.lua`, the default value for this is going to be a function. Then, because of how the options are merged, it will become impossible for it to ever be `nil`. It would be nice to still let the user disable it by setting it to `false`, so maybe replace this with `not config.post_parse_symbol`